c9s.repo

```diff
@@ -1,4 +1,4 @@
-[baseos]
+[c9s-baseos]
```

Comment:
From memory that breaks the extension container builds

Reply:
Seems to work fine locally. It's also tested in CI here. Possibly whatever broke on it before is now fixed?